# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/style/FontFaceSetImpl.cpp
# Commit: 609a5d93bf08
# Full Hash: 609a5d93bf08d65f6a5eafc4c0d20290d23dacd5
# Author: Andrew Osmond <aosmond@mozilla.com>
# Date: 2022-06-29 03:44:30
# Regressor Bug: 1771493
# File Overlap Count: 1
# Description:
#   Bug 1771493 - Part 3. Abstract away buffer extraction from FontFaceImpl::InitializeSource. r=emilio
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D147818
# ==============================================================================

diff -r 1f5b0b087930 -r 609a5d93bf08 layout/style/FontFaceSetImpl.cpp
--- a/layout/style/FontFaceSetImpl.cpp	Tue Jun 28 15:58:10 2022 +0000
+++ b/layout/style/FontFaceSetImpl.cpp	Tue Jun 28 15:58:10 2022 +0000
@@ -878,7 +878,7 @@
     }
 
     face->mSourceType = gfxFontFaceSrc::eSourceType_Buffer;
-    face->mBuffer = aFontFace->CreateBufferSource();
+    face->mBuffer = aFontFace->TakeBufferSource();
   } else {
     AutoTArray<StyleFontFaceSourceListComponent, 8> sourceListComponents;
     aFontFace->GetSources(sourceListComponents);
